src/sentry/south_migrations/0477_incident_group.py

```diff
@@ -0,0 +1,1390 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding model 'IncidentGroup'
+        db.create_table('sentry_incidentgroup', (
```

Comment:
@mattrobenolt Could I get a blessing on this? Do I actually need a blessing for adding tables that don't link to any existing core tables?

Reply:
@mattrobenolt Post pycon bump

Reply:
Nah, this is fine.